integration-tests/plugins/__tests__/api-key/admin/api-key.spec.ts

```diff
@@ -106,4 +106,29 @@ describe("API Keys - Admin", () => {
     expect(deleted.status).toEqual(200)
     expect(listedApiKeys.data.apiKeys).toHaveLength(0)
   })
+
+  it.skip("can use a secret api key for authentication", async () => {
```

Comment:
I am waiting for #6447 to get merged first before we can actually enforce the API keys. I'll add the enforcement changes there